web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1321,10 +1321,29 @@ def print_reports(client,
     if output_dir and not os.path.exists(output_dir):
         os.makedirs(output_dir)
 
+    if basename_local_dirs:
+        LOG.info("Matching local report directories (--baseline): %s",
+                 basename_local_dirs)
+    if newname_local_dirs:
+        LOG.info("Matching local report directories (--newname): %s",
+                 newname_local_dirs)
+
     client = None
     # We set up the client if we are not comparing two local report directory.
     if basename_run_names or newname_run_names:
-        client = setup_client(args.product_url)
+        try:
+            client = setup_client(args.product_url)
+        except SystemExit as sexit:
+            LOG.info("Could not connect to remote server uri=%s. "
+                     "Check if basenames and newnames are valid directory "
+                     "names or remote if accessible by URI.",
+                     args.product_url)
+            raise sexit
+
+    if basename_run_names:
+        LOG.info("Matching remote runs (--baseline): %s", basename_run_names)
+    if newname_run_names:
+        LOG.info("Matching remote runs (--newname): %s", newname_run_names)
```

Comment:
Lets print these information above before creating a connection towards the server:
```suggestion
```